CIP-????/README.md

```diff
@@ -0,0 +1,474 @@
+state projection functions `πˢ, πˢ'`, and `πⁱ = id` for both, we get a new structured contract `STRUC''`.
+It has the projection function `<πˢ, πˢ'>` which pairs the two states, `πⁱ = id`, and the specification 
+
+`{ ((a, a'), (b, b'), (c, c'), (d, d')) ∈ STRUC'' | (a, b, c, d) ∈ STRUC AND (a', b', c', d') ∈ STRUC'}`
+
+This allows reasoning about contract interaction. For example, each account in the account simulation 
+can be viewed as an individual stateful program, and composing all accounts together in this way 
+also defines a stateful program. Such a composition allows us to reason about account transfers, which 
+requires the interaction of at least two accounts. 
+
+## Rationale: How does this CIP achieve its goals?
+
+This CIP defines a framework for building stateful programs running on the ledger, implemented in a 
+way that guarantees that their small-steps specifications are never violated. The examples 
+and discussion we give in this CIP should hopefully convince the reader that 
+
+- structured contracts are the right approach for reasoning about specifying and implementing  
+any stateful program on the ledger, and
+
+- all script verification should be viewed as an effort to ensure that some corresponding stateful program 
+is an instance of the right structured contract. That is, one that satisfies some properties desired by 
+the author, and describes the evolution of the the relevant data on the ledger.
+
+We provide the specification and set of examples that must be implemented as part of this CIP
+to demonstrate the above statements, and give users the tools to implement their own formally 
+verified stateful programs on the Cardano ledger.
+
+### Automation Potential
+
+The ideal eventual outcome of this script verification effort is automation of implementing 
+a given program specification. However, at this time, there is no specific plan for
+how to achieve this goal in the general case. However, building specific structured 
+contract templates for common design patters will make this framework much more 
+user-friend and accessible.
+
+## Comparison with Other Designs
+
+The most advanced smart contract verification effort is the 
+[SmartCodeVerifier: Automated Formal Verification Tool for Smart Contract Code](https://cardano.ideascale.com/c/cardano/idea/132861).
+Both efforts can coexist alongside each other, filling different needs.
+
+Our proposal has the advantage that it is build on a full Cardano ledger model, 
+ensuring that everything scripts can do on the real Cardano ledger can be encoded and 
+structured contract instances in our framework. The `SmartCodeVerifier` 
+is written in Lean, and is associated with a reduced ledger model. Therefore, it is  
+less versatile. However, 
+it does support far better push-button verification automation.
+
+
+
+## Path to Active
+
+### Acceptance Criteria
+
+Two to four of the examples/usecases are implemented. 
+
+
+### Implementation Plan
+
+1. Instantiate `Value`
```

Comment:
According to https://github.com/cardano-foundation/CIPs/pull/963#discussion_r1916602838 & further review / development it will ultimately help to either drop this item or elaborate upon it... but should be fine to leave as-is in this preliminary stage.

Reply:
thank you for all the comments! will make the changes shortly

Reply:
Section suggestions done

Reply:
thanks & enough clarification for the Ledger dependency seems provided by https://github.com/cardano-foundation/CIPs/pull/963/commits/144c0181ac62c3a5fe8c350d4ba2f7c1e4a9056b.